mithril-stm/src/aggregate_signature/mod.rs

```diff
@@ -3,19 +3,21 @@ mod clerk;
 mod core_verifier;
 mod signature;
 
-pub use aggregate_key::*;
-pub use clerk::*;
-pub use core_verifier::*;
+pub use aggregate_key::StmAggrVerificationKey;
+pub use clerk::StmClerk;
+pub use core_verifier::CoreVerifier;
```

Comment:
Maybe we can systematically use a wildcard `*` in the exports of internal sub-modules of the crate?